packages/playground/_stories/fiori/MediaGallery/MediaGallery.stories.ts

```diff
@@ -60,230 +82,235 @@ Basic.args = {
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-1000.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-1010.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-1022.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-1030.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-2002.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-2026.jpg" />
-            </ui5-media-gallery-item>
-        </ui5-media-gallery>
-    </div>
-`;
+export const WithVerticalLayout = Template.bind({});
+WithVerticalLayout.args = {
+    showAllThumbnails: true,
+    layout: MediaGalleryLayout.Vertical,
+    default: `
+    <ui5-media-gallery-item>
+        <img src="../assets/images/HT-1000.jpg" />
+    </ui5-media-gallery-item>
+    <ui5-media-gallery-item>
+        <img src="../assets/images/HT-1010.jpg" />
+    </ui5-media-gallery-item>
+    <ui5-media-gallery-item>
+        <img src="../assets/images/HT-1022.jpg" />
+    </ui5-media-gallery-item>
+    <ui5-media-gallery-item>
+        <img src="../assets/images/HT-1030.jpg" />
+    </ui5-media-gallery-item>
+    <ui5-media-gallery-item>
+        <img src="../assets/images/HT-2002.jpg" />
+    </ui5-media-gallery-item>
+    <ui5-media-gallery-item>
+        <img src="../assets/images/HT-2026.jpg" />
+    </ui5-media-gallery-item>`,
+};
 
-export const Template2: StoryFn = () => html`
-    <h3>MediaGallery with thumbnails on the right</h3>
-    <div class="snippet">
-        <ui5-media-gallery
-            show-all-thumbnails=""
-            layout="Horizontal"
-            menu-horizontal-align="Right"
-        >
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-1000.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-1010.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-1022.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-1030.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-2002.jpg" />
-            </ui5-media-gallery-item>
-            <ui5-media-gallery-item>
-                <img src="../assets/images/HT-2026.jpg" />
-            </ui5-media-gallery-item>
-        </ui5-media-gallery>
-    </div>
-`;
+export const WithInteractiveDisplayArea = Template.bind({});
+WithInteractiveDisplayArea.args = {
```

Comment:
This sample is missing interactive-display-area attribute and doesn't show this feature

Reply:
Actually, the name of the story was incorrect. The correct name should be `With Thumbnails On The Right`.
Thanks, resolved.